app/javascript/components/vm-common-rename-form/index.jsx

```diff
@@ -0,0 +1,67 @@
+import React, { useState, useEffect } from 'react';
+import MiqFormRenderer from '@@ddf';
+import PropTypes from 'prop-types';
+import createSchema from './vm-common-rename-form.schema';
+import { API } from '../../http_api';
+import miqRedirectBack from '../../helpers/miq-redirect-back';
+
+const VmCommonRenameForm = ({ vmId }) => {
+  const [{ initialValues, fields }, setState] = useState({
+    isLoading: !!vmId,
+    fields: [],
+  });
+
+  useEffect(() => {
+    if (vmId) {
+      API.get(`/api/vms/${vmId}`).then((initialValues) => {
+        setState((state) => ({ ...state, initialValues, isLoading: false }));
```

Comment:
You are setting isLoading true/false, but not using that variable anywhere . Please refer other forms on usgae of it, let me know if you need any help.